### PR TITLE
Revert "tests: stop using ValuesFile, use ControlPlaneValues"

### PR DESCRIPTION
**Pull request by howardjohn**
Reverts istio/istio#21065

I don't know why, but this broke security tests. The correct move here should be revert then investigate rather than leave things broken for days

### CODE REVIEW

**Comment by howardjohn** [comment]
ref https://github.com/istio/istio/issues/21149

File: pkg/test/framework/components/istio/config.go [hunk 1]
```diff
@@ -16,14 +16,20 @@ package istio
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 	"strings"
 	"time"
 
+	"github.com/mitchellh/go-homedir"
```
Comment (fpesce) [inline pkg/test/framework/components/istio/config.go]: What is this? Don't golang has os.UserHomeDir that could be used since 1.12? ;-)

Comment (howardjohn) [reply pkg/test/framework/components/istio/config.go]: I am reverting a PR, I didn't touch any code here...

**Comment by dgn** [comment]
I'm not sure I follow. @howardjohn the test failure you sent me on slack is from a postsubmit run that was started 1h before my PR was merged.

**Comment by howardjohn** [comment]
take  look at https://github.com/istio/istio/commits/master, you can see it failed on all PRs until this was merged

related - https://github.com/istio/istio/issues/21149

**Comment by dgn** [comment]
I see- nevermind then. Will keep investigating and come back to it. Thank you!

**Comment by howardjohn** [comment]
Thanks! sorry for the revert, I just didn't want to leave us in a broken
state now that we generally have tests always passing in postsubmit now. I
am bringing up in the T&R meeting that we should probably not have
postsubmit only tests since these types of issues are so easily avoidable
by just running everything in presubmit

**Comment by dgn** [comment]
Don't worry! I broke master, reverting was the right thing to do :) I'm actually glad you found out early; I am still not 100% sure why this is happening but I will make sure to find out on Monday and provide an updated PR. I fully agree that postsubmits should equal presubmits, I might just submit a PR to test-infra on Monday as well.

Have a nice weekend!

**Comment by howardjohn** [comment]
The reason tests are postsubmit are because they are either flaky (this is
mostly not true anymore, everything is fairly stable) or long. If we just
flip those to presubmit the test will be over 1hr long, so we need to
either split it into 2 suites or optimize the tests first I think. 1 hr is
a bit long

**Pull request merged**